jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/PathSpec.java

```diff
@@ -85,6 +85,13 @@
      */
     String getSuffix();
 
+    /**
+     *  Test if a normalized string pathspec is equivalent to this pathspec
```

Comment:
This makes it sound as if the caller is expected to pass in a string that has already had normalize() called on it
```suggestion
     *  Test if a string pathspec is equivalent to this pathspec.
```